ast.go

```diff
@@ -450,6 +450,20 @@ func unaryOp(tok token.Token, args []*internal.Elem) constant.Value {
 func binaryOp(cb *CodeBuilder, tok token.Token, args []*internal.Elem) constant.Value {
 	if len(args) == 2 {
 		if a, b := args[0].CVal, args[1].CVal; a != nil && b != nil {
+			if tok == token.QUO {
+				switch b.Kind() {
```

Comment:
introduce `func isZero(v constant.Value) bool` and don't use v.String() to compare with zero.

Reply:
maybe can use https://pkg.go.dev/go/constant#Sign

Reply:
add func checkDivisionByZero use constant.Sign